src/easylogging++.h

```diff
@@ -2439,7 +2439,7 @@ class LogFormat : public Loggable {
 };
 }  // namespace base
 /// @brief Resolving function for format specifier
-typedef std::function<const char*(void)> FormatSpecifierValueResolver;
+typedef std::function<const char*(const LogMessage&)> FormatSpecifierValueResolver;
```

Comment:
Please pass in `const LogMessage*` instead as we already have logMessage as pointer in line 4632 below